src/main/java/janggi/view/OutputView.java

```diff
@@ -0,0 +1,111 @@
+package janggi.view;
+
+import janggi.piece.PieceType;
+import janggi.piece.Team;
+import janggi.piece.pieces.Piece;
+import janggi.position.Position;
+import janggi.position.Route;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OutputView {
+    private static final int MAX_ROW = 10;
+    private static final int MAX_COLUM = 9;
```

Comment:
```suggestion
    private static final int MAX_COLUMN = 9;
```

Position에도 유사한 상수가 정의되어 있네요.